Formula/global.rb

```diff
@@ -36,21 +35,12 @@ class Global < Formula
   depends_on "libtool"
   depends_on "ncurses"
   depends_on "python@3.10"
-  depends_on "sqlite"
```

Comment:
I don't think we want to do this, since it will vendor its own otherwise.

See https://github.com/Homebrew/homebrew-core/pull/81424.

Reply:
I asked to remove it because @chenrui333 removed the patch from the `pygments` resource, so I thought the issue was solved and it did not need `sqlite` anymore.
Does the patch still apply? If not and it's still needed, we need a new patch. But we should also open an issue upstream to add a way to use an external `sqlite`.

Reply:
The patch has been upstreamed and accepted (see https://github.com/Homebrew/homebrew-core/pull/81424). Not sure if it's in this version.